ui/components/Popup.js

```diff
@@ -102,18 +116,22 @@ function Popup() {
             className={classes.paper}
           >
 
-            <Typography className={classes.header} variant="h6" >Get early access to MeshMap!
-            </Typography>
+            <div className={classes.headerWrapper}>
+              <Typography className={classes.header} variant="h6" >Get early access to MeshMap!
+              </Typography>
 
-            <div style={{ display : "flex", justifyContent : "flex-end", whiteSpace : "nowrap", position : "relative" }}>
-              <IconButton  key="close" aria-label="Close" color="inherit" onClick={() => setIsOpen(false)}>
-                <CloseIcon />
-              </IconButton>
+              <div style={{ display : "flex", justifyContent : "flex-end", whiteSpace : "nowrap", position : "relative" }}>
+                <IconButton  key="close" aria-label="Close" color="inherit" onClick={() => setIsOpen(false)}>
+                  <CloseIcon />
+                </IconButton>
+              </div>
+            </div>
 
+            <div className={classes.imgWrapper}>
+              <img className={classes.drawer} src="/static/img/designer-drawer.png" />
+              <img className={classes.canvas} src="/static/img/designer-canvas.png" />
             </div>
-            <img className={classes.drawer} src="/static/img/designer-drawer.png" />
-            <img className={classes.canvas} src="/static/img/designer-canvas.png" />
-            <Typography variant="subtitle1"><i>Friends dont let friends GitOps alone. Visually design and collaborate in real-time with other MeshMap users.</i></Typography>
+            <Typography className={classes.caption} variant="subtitle1"><i>Friends dont let friends GitOps alone. Visually design and collaborate in real-time with other MeshMap users.</i></Typography>
```

Comment:
btw there is a variant called "caption" also, maybe it wasn't useful here?

Reply:
@Abhishek-kumar09 what do you mean by "variant"? the className "caption" has css for styling the text. removing it would mean adding inline css which isn't ideal. please clarify

Reply:
Hi @Abhishek-kumar09 I have also attached a screenshot of the completed work below
![Uploading Screenshot 2022-11-14 at 12.38.52.png…]()

Reply:
<img width="529" alt="Screenshot 2022-11-14 at 12 38 52" src="https://user-images.githubusercontent.com/30264870/201653494-ca6ca4dc-5278-4e92-a274-42e321ca68f3.png">

Reply:
Ok